binding.gyp

```diff
@@ -18,6 +18,16 @@
       'dependencies': [
         'deps/libmagic/libmagic.gyp:libmagic',
       ],
+      'conditions': [
+        ['OS=="mac"', {
+          'xcode_settings': {
+            'MACOSX_DEPLOYMENT_TARGET': '10.9',
```

Comment:
I think we should set this to 10.7 to match what node currently uses.

Reply:
10.7 throws a deprecation warning. Is it OK?

Reply:
What deprecation warning?

Reply:
No warning :) My fault.